content/blog/kubecon-2023.md

```diff
@@ -0,0 +1,54 @@
+---
+id: blog
+title: Kubecon, is it still for me?
+description: Lee looks back on his second Kubecon, lists some standout-talks and discusses his takeaway.
+summary: Lee looks back on his second Kubecon, lists some standout-talks and discusses his takeaway.
+author: Lee Beenen
+tags:
+  - cloud
+  - kubernetes
+  - k8s
+  - cloud native security
+  - vulnerability scanning
+slug: kubecon-2023
+date: 2023-05-05
+---
+Kubecon 2023 is a wrap! In fact, it's already a couple of weeks in the past. As only my second Kubecon ever (Kubecon Valencia 2022 was my first), it leaves me with a lot of mixed feelings. 
+
+Yes, it was a well run event, it was filled with interesting talks and like last year, I learned about a lot of stuff happening outside the bubble of my current assignment, which is one of the main attractions of being a consultant at Skyworkz, and going to a tech conference to meet people.
+
+However, KubeCon also left me thinking that I am no longer its target audience. Why? 
+
+We'll get to that, let's first lead off with couple of stand-out talks that I feel are worth a (re)-watch.
+
+## 1.  Enabling Real-Time Media in Kubernetes, Giles Heron
+https://youtu.be/1_cxXzhY4Xg
```

Comment:
This should be a link, or you can try something like this below, that _should_ potentially embed the video in the blog, but I'm not sure if we have that enabled for the Skyworkz blog or not 😉  

```markdown
{{< youtube 1_cxXzhY4Xg >}}
```

Reply:
It works, but the embed is rather...large. It's more obvious than a clickable header though, so I'll go with that. Thanks!